internal/cmd/connect/utils.go

```diff
@@ -20,9 +25,15 @@ func getConfig(cmd *cobra.Command) (*map[string]string, error) {
 		return nil, errors.Wrapf(err, errors.UnableToReadConfigurationFileErrorMsg, configFile)
 	}
 
+	connectorType := options["confluent.connector.type"]
+	if connectorType != "" {
+		connectorType = "MANAGED"
+	}
+
 	_, nameExists := options["name"]
 	_, classExists := options["connector.class"]
-	if !nameExists || !classExists {
+
+	if connectorType != "CUSTOM" && (!nameExists || !classExists) {
```

Comment:
`connectorType != "CUSTOM"` will always evaluate to true. Based on the logic above, `connectorType` should either be `""` or `"Managed"`.

Reply:
Good catch. Fixed it. Thanks!